lib/server/formations.ts

```diff
@@ -96,15 +38,20 @@ async function _getFormation(id: string): Promise<FormationData | false> {
   return buildFormationJson(formation, user);
 }
 
-export const getFormation = cache(_getFormation);
+export const getFormation = unstable_cache(_getFormation, [], {
```

Comment:
wait, so if the user edits a formation... the cache only revalidates every hour? so how would they see the new formation edited?

Reply:
in the `updateFormation` function we call `revalidatePath` with the formation's path URL in order to update the data